plugins/woocommerce/src/StoreApi/Schemas/V1/CartSchema.php

```diff
@@ -342,8 +343,12 @@ public function get_item_response( $cart ) {
 		// calculated so we can avoid returning costs and rates prematurely.
 		$has_calculated_shipping = $cart->show_shipping();
 
-		// Get shipping packages to return in the response from the cart.
-		$shipping_packages = $has_calculated_shipping ? $controller->get_shipping_packages() : [];
+		$has_local_pickup_methods = LocalPickupUtils::is_local_pickup_enabled() && count( LocalPickupUtils::get_local_pickup_method_ids() ) > 0;
+
+		// Get shipping packages to return in the response from the cart. Always get the shipping packages if local
+		// pickup is enabled and has methods. If the address is required then regular shipping rates will be filtered
+		// out later.
+		$shipping_packages = $has_calculated_shipping || $has_local_pickup_methods ? $controller->get_shipping_packages() : [];
```

Comment:
If there are local pickup methods, we should return these regardless of whether an address has been entered. `$has_calculated_shipping` is a _little_ misleading, as `$cart->show_shipping()` checks whether shipping is enabled and cart has items, and if an address is required to show shipping, returns whether the address fields are complete, otherwise returns true, so it's not really to do with calcuation completeness, but address completeness.